crates/aptos/debug-move-example/Move.toml

```diff
@@ -7,4 +7,3 @@ DebugDemo = "0x1"
 
 [dependencies]
 AptosFramework = { local = "../../../aptos-move/framework/aptos-framework" }
-MoveNursery = {local = "../../../aptos-move/framework/move-stdlib/nursery"}
```

Comment:
Why delete this?

Reply:
The nursery had been removed from our repo a while ago, so this wasn't compiling any longer. Drive by effect as I used it as a test case. Nothing you should be concerned about.